.github/workflows/propagate_to_manifestdb.yml

```diff
@@ -0,0 +1,77 @@
+name: Propagate update to OSBuild
```

Comment:
I think that this `name` is misleading, it should be something like "Update osbuild ref in manifest-db"